src/WCSimDetectorConfigs.cc

```diff
@@ -410,18 +410,18 @@ void WCSimDetectorConstruction::SetNuPrism_mPMTGeometry()
 	vessel_cyl_height = 38.*CLHEP::mm;    //option A, option B would be 277 mm
 	vessel_radius_curv = 342.*CLHEP::mm;  //needs to include the vessel thickness, as we construct from outside inwards.
 	vessel_radius = 254.*CLHEP::mm;
-	dist_pmt_vessel = 8*CLHEP::mm;      
+	dist_pmt_vessel = 2*CLHEP::mm;      
 	orientation = PERPENDICULAR;
 	mPMT_outer_material = "G4_PLEXIGLASS";
 	mPMT_inner_material = "Air";               // TODO: real air, hence update abs_length
 	mPMT_material_pmtAssembly = "SilGel";
 	mPMT_outer_material_d = 10*CLHEP::mm;
 	
 	// Radius of cone at z=reflectorHeight
-	id_reflector_height = 9.29*CLHEP::mm;        // From solidworks CAD drawings
-	id_reflector_z_offset = 6.12*CLHEP::mm;
-	id_reflector_angle = 46.5*CLHEP::deg;
-	mPMT_pmt_openingAngle = 8.3*CLHEP::deg;
+	id_reflector_height = 6.53*CLHEP::mm;        // for a radius of 7.25mm, for hex: 5.4mm (radius of 6mm)
```

Comment:
Have these changes been documented and studied somewhere? This will affect the IWCD model as well, right?

Reply:
I need to double check this... I thought they would be the same too. The comment on L 421 isn't mine and I don't think I changed this

Reply:
These changes seem to be reverting to an older version of the mPMT design. See [this commit](https://github.com/WCTE/WCSim/commit/886705c9b862acfed8438a60d0d790d90c1abdca).

Reply:
Looks like the mPMT design got reverted back in [this commit](https://github.com/laurenanthony2/WCSim/commit/e13bdd44573923cb63a54951344cd9ee811908a0) by Lauren where she added the WCTE designs. I guess the WCTE design was developed using an old version of this file.

Reply:
I think Nick is right. I thought I'd updated my version. I'll get the newest version of detector configs and add my changes there.